### ISSUE TITLE
The latest version has a broken go.mod file

### ISSUE THREAD

**Issue by arjenvdhave** [arch-go/arch-go#103]
### Preflight checklist

- [X] I agree to follow this project's [Code of Conduct](https://github.com/arch-go/arch-go/blob/main/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/arch-go/arch-go/blob/main/CONTRIBUTING.md).
- [X] I could not find a solution in the existing [issues](https://github.com/arch-go/arch-go/issues) or [discussions](https://github.com/arch-go/arch-go/discussions).

### Describe the bug

When running `go install -v github.com/arch-go/arch-go@latest` it will download version 1.5.4
In this version the go.mod file has the old repo name.

Resulting in an error:
```
go: downloading github.com/arch-go/arch-go v1.5.4
go: github.com/arch-go/arch-go@latest: version constraints conflict:
        github.com/arch-go/arch-go@v1.5.4: parsing go.mod:
        module declares its path as: github.com/fdaines/arch-go
                but was required as: github.com/arch-go/arch-go
```

### How can the bug be reproduced

Run `go install -v github.com/arch-go/arch-go@latest`

### Relevant log output

```shell
go: downloading github.com/arch-go/arch-go v1.5.4
go: github.com/arch-go/arch-go@latest: version constraints conflict:
        github.com/arch-go/arch-go@v1.5.4: parsing go.mod:
        module declares its path as: github.com/fdaines/arch-go
                but was required as: github.com/arch-go/arch-go
```
```


### Relevant configuration

_No response_

### Version

1.5.4

### Additional Context

_No response_

**Comment by DGuhr** [comment]
Hey @seckinsen, @igoramorim and @arjenvdhave - First of all, thanks for noticing and filing the issue. We switched the repo lately, which led to this. if you install fdaines/arch-go it would work, but we're currently fixing this and will release a new version shortly that is working via go install again. For the future, we plan a few more changes and releasing a new major version that might also include some breaking ones, see the other issues for details.

**Comment by dadrus** [comment]
Resolved with the new 1.6.0, respectively 1.6.1 release

**Comment by DGuhr** [comment]
jfyi, we found another small bug in #110 and are currently releasing a v1.6.2 - should be there now if you install the latest version.
